RiotSharpTest/TournamentRiotApiTest.cs

```diff
@@ -114,5 +114,157 @@ public void GetTournamentMatchAsync_Test()
             Assert.AreEqual(Season.PreSeason2016, details.Season);
             Assert.AreEqual("5.24.0.256", details.MatchVersion);
         }
+
+        #region CreateTournamentCodes
+       
+        [TestMethod]
+        [TestCategory("TournamentRiotApi")]
+        public void CreateTournamentCodes_InvalidTeamSize_ThrowsArgumentException()
+        {
+            try
+            {
+                // Act 
+                var tournamentCodes = api.CreateTournamentCodes(0, 0, TournamentSpectatorType.All, TournamentPickType.TournamentDraft, TournamentMapType.SummonersRift);
+            }
+            catch (ArgumentException e)
+            {
+                // Assert
+                Assert.IsInstanceOfType(e, typeof(ArgumentException));
+                Assert.AreEqual("teamSize", e.ParamName);
+            }
+        }
+         
+        [TestMethod]
+        [TestCategory("TournamentRiotApi"), TestCategory("Async")]
+        public void CreateTournamentCodesAsync_InvalidTeamSize_ThrowsArgumentException()
+        {
+            try
+            {
+                // Act 
+                var tournamentCodes = api.CreateTournamentCodesAsync(0, 0, TournamentSpectatorType.All, TournamentPickType.TournamentDraft, TournamentMapType.SummonersRift).Result;
+            }
+            catch(AggregateException e)
+            {
+                // Assert
+                Assert.IsInstanceOfType(e, typeof(AggregateException));
+                Assert.IsInstanceOfType(e.InnerException, typeof(ArgumentException));
+                var argumentException = (ArgumentException)e.InnerException;
+                Assert.AreEqual("teamSize", argumentException.ParamName);
+            }
+        }
+
+        [TestMethod]
+        [TestCategory("TournamentRiotApi")]
+        public void CreateTournamentCodes_CountGreaterThan100_ThrowsArgumentException()
+        {
+            try
+            {
+                // Act 
+                var tournamentCodes = api.CreateTournamentCodes(0, 5, TournamentSpectatorType.All, TournamentPickType.TournamentDraft, TournamentMapType.SummonersRift, count: 1001);
+            }
+            catch (ArgumentException e)
+            {
+                // Assert
+                Assert.IsInstanceOfType(e, typeof(ArgumentException));
+                Assert.AreEqual("count", e.ParamName);
+            }
+        }
+
+        [TestMethod]
+        [TestCategory("TournamentRiotApi"), TestCategory("Async")]
+        public void CreateTournamentCodesAsync_CountGreaterThan100_ThrowsArgumentException()
+        {
+            try
+            {
+                // Act 
+                var tournamentCodes = api.CreateTournamentCodesAsync(0, 5, TournamentSpectatorType.All, TournamentPickType.TournamentDraft, TournamentMapType.SummonersRift, count: 1001).Result;
```

Comment:
could you reformat those lines so that they are less than 100 or 120 chars, it'll be easier to read?